eZ/Bundle/EzPublishRestBundle/Tests/Functional/ViewTest.php

```diff
@@ -8,76 +8,179 @@
  */
 namespace eZ\Bundle\EzPublishRestBundle\Tests\Functional;
 
+use stdClass;
+
 class ViewTest extends TestCase
 {
+    /** @var array */
```

Comment:
Can we add information about the type hint of the ID?

Reply:
Fixed via c6ea2de.